include/vcpkg/base/format.h

```diff
@@ -7,76 +7,12 @@
 #include <vcpkg/base/pragmas.h>
 #include <vcpkg/base/stringview.h>
 
-namespace vcpkg
+template<class Char>
+struct fmt::formatter<std::error_code, Char> : formatter<std::string, Char>
```

Comment:
FYI this will go away if/when I finally finish https://github.com/BillyONeal/vcpkg-tool/tree/expected-filesystem :/